ubuntu_server_triage.py

```diff
@@ -132,13 +164,15 @@ def main(start, end=None):
 
     connect_launchpad()
     logging.info('Ubuntu Server Bug List')
-    bugs = create_bug_list(start, end)
-    print_bugs(bugs)
+    report_current_backlog(lpname)
+    bugs = create_bug_list(start, end, lpname, bugsubscriber)
+    print_bugs(bugs, open_in_browser)
 
 
 if __name__ == '__main__':
     PARSER = argparse.ArgumentParser()
     PARSER.add_argument('start_date',
+                        nargs='?',
```

Comment:
I'm mixed on this one. I love the ability to search for all bugs, however as the "default" behavior with no args I hesitate as it leads to a very, very long execution time. 

What do you think about:
1) if no arguments, search for all the bugs modified today or yesterday
2) if the -a --all argument is provided, dates are ignored and we look for all bugs?

This way the default behavior would be faster and more useful?